internal/output/html/vuln_table_template.html

```diff
@@ -0,0 +1,34 @@
+<table class="vuln-table">
+  <tr>
+    <th>ID</th>
+    <th>package</th>
+    <th>Installed version</th>
+    <th>Fixed version</th>
+    <th>Severity</th>
+  </tr>
+  {{ range $rowIndex, $element := . }}
+  {{ $index := uniqueID}}
+  <tr class='vuln-tr' id="vuln-tr-{{ $element.Summary.ID }}-{{ $rowIndex }}-{{ $index }}"
+    onclick="showVulnDetails('{{ $element.Summary.ID }}-{{ $rowIndex }}-{{ $index }}')">
+    <td><a href="https://osv.dev/{{ $element.Summary.ID }}">{{ $element.Summary.ID }}</a></td>
+    <td>{{ $element.Summary.PackageName }}</td>
+    <td>{{ $element.Summary.InstalledVersion }}</td>
+    <td {{ if eq $element.Summary.FixedVersion "No fix available" }} class="warning" {{ end }}>{{
+      $element.Summary.FixedVersion }}</td>
+    <td>{{ $element.Summary.SeverityRating }} {{ if ne $element.Summary.SeverityRating "UNKNOWN" }}({{
+      $element.Summary.SeverityScore }}){{ end }}</td>
```

Comment:
nit: Fix the line breaks here so that {{ and }} are in the same line.

Reply:
Updated, I feel it got messed up by the reformat script